recipes/bluesky-queueserver-api/meta.yaml

```
@@ -0,0 +1,45 @@
+{% set name = "bluesky-queueserver-api" %}
+{% set version = "0.0.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/bluesky-queueserver-api-{{ version }}.tar.gz
+  sha256: df7e2801e80f6c5aa006dc30c390705b5642f8debea1e63e84aedb8e57beb4a5
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - pip
+    - python >=3.7
+  run:
+    - bluesky-queueserver
+    - httpx
+    - python >=3.7
+
+test:
+  imports:
+    - bluesky_queueserver_api
+  commands:
+    - pip check
+  requires:
+    - pip
```

As the `pip check` fails for Windows, let's exclude this check for that platform.
```suggestion
  commands:      # [not win]
    - pip check  # [not win]
  requires:      # [not win]
    - pip        # [not win]
```